package.json

```diff
@@ -56,6 +56,7 @@
   "dependencies": {
     "alasql": "^4.1.9",
     "axios": "^1.6.2",
+    "body-parser": "^1.20.2",
```

Comment:
why busboy is not enough?

Reply:
check this https://github.com/mscdex/busboy/issues/125; busboy dont support application/json content-type